schema/responses.proto

```diff
@@ -80,4 +80,5 @@ message QueryResponse {
         RolesResponse roles_response = 7;
         RolePermissionsResponse role_permissions_response = 8;
     }
+    bytes query_hash = 9;
```

Comment:
I guess, more convenient to name as just `hash` because of the context `Query` already.